pkgs/servers/http/nginx/modules.nix

```diff
@@ -370,6 +370,16 @@ in
     };
   };
 
+  spnego-http-auth = {
+    src = fetchFromGitHub {
+      name = "spnego-http-auth-nginx-module";
```

Comment:
Can rename to `name = "spnego-http-auth";`

Reply:
Done ✔️